valheim-backup

```diff
@@ -49,6 +51,11 @@ backup() {
     chmod "$BACKUPS_FILE_PERMISSIONS" "$backup_file"
 }
 
+rsync_to_sftp() {
```

Comment:
Maybe I missed it, but where does it check if `BACKUPS_SFTP_ENABLE=true`? Wouldn't it execute `rsync` even with `BACKUPS_SFTP_ENABLE=false`?

Reply:
*facepalm* big oof, i knew it was to late yesterday. Sorry